tests/README.md

```diff
@@ -397,6 +409,9 @@ ali:
     # optional, path in bucket 
     bucket_path: integration-test
 
+    # port for remote connection (required)
+    port: 22
+
```

Comment:
Won't this break existing tests? In line 376 you write "default: 22" and here it is a required parameter.

Reply:
Good catch; it will. Therefore, I removed this again and it's only left within the KVM platform. However, 'sshclient.py' will now set it to tcp/22 as default if not overwritten by any other functions (currently only in KVM). This change won't blow up the config(s) (examples) and still keeps the possibility to include this in 'RemoteClient' if any cloud platform would need this in future.

jftr; not yet pushed.